mets_mods2tei/api/tei.py

```diff
@@ -277,7 +277,7 @@ def digital_editions(self):
         Return information on the editions of the digitalized work represented
         by the TEI Header.
         """
-        return [digital_edition.text for digital_edition in self.tree.xpath('//tei:fileDesc/tei:titleStmt/tei:editionStmt/tei:edition', namespaces=ns)]
+        return [digital_edition.text for digital_edition in self.tree.xpath('//tei:fileDesc/tei:editionStmt/tei:edition', namespaces=ns)]
```

Comment:
Is this a fix of your own modifications? (I am somewhat lost in this mega PR.) If not I am surprised that I introduced such a dramatic error. Will consult DTABf...

Reply:
No, the error is all yours :-) (I was surprised, too.)

It was already invalid in TEI proper.